tests/test_js_utils.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from tests.conftest import BaseGraphCase
+from tests.conftest import GraphCaseFactory
+
+
+@pytest.fixture
+def graph(graph_cases: GraphCaseFactory) -> BaseGraphCase:
+    return graph_cases('empty')
+
+
+def test_set_style_key(graph: BaseGraphCase) -> None:
+    assert graph.eval_js_function('graphs.utils.setStyleKey', '', 'bar', 5) == 'bar=5'
+    assert graph.eval_js_function('graphs.utils.setStyleKey', 'style', 'bar', 5) == 'style;bar=5'
+    assert (
+        graph.eval_js_function('graphs.utils.setStyleKey', 'style;bar=7', 'bar', 5) == 'style;bar=5'
+    )
+    assert (
+        graph.eval_js_function('graphs.utils.setStyleKey', 'foobar=3;bar=7', 'bar', 5)
+        == 'foobar=3;bar=5'
+    )
+    assert (
+        graph.eval_js_function('graphs.utils.setStyleKey', 'foobar=3', 'bar', 5) == 'foobar=3;bar=5'
+    )
+    assert (
```

Comment:
Not a problem.
Don't need to change.
A formatting nitpick.

I think that using a `partial` will prevent precommit hook from breaking those lines.
```python
    set_style_key = partial(graph.eval_js_function, 'graphs.utils.setStyleKey')
    assert set_style_key('foobar=3;fizzbar=7', 'bar', 5) == 'foobar=3;fizzbar=7;bar=5'
```

Reply:
Is there a JS autoformatter we can use with pre-commit?

EDIT: https://prettier.io?

Reply:
https://prettier.io/docs/en/precommit.html#option-3-pre-commithttpsgithubcompre-commitpre-commit

Reply:
https://prettier.io/docs/en/configuration.html#basic-configuration

Reply:
@oliveira-mauricio don't worry about doing it in this PR though, we can add the configuration in a follow up.

Reply:
Sure, I will create a task to tackle that later.